python/kwiver/vital/types/camera_perspective.cxx

```diff
@@ -91,7 +93,9 @@ PYBIND11_MODULE( camera_perspective, m )
   .def( "set_translation",  &kv::simple_camera_perspective::set_translation )
   .def( "set_center_covar", &kv::simple_camera_perspective::set_center_covar )
   .def( "set_rotation",     &kv::simple_camera_perspective::set_rotation )
-  .def( "set_intrinsics",   &kv::simple_camera_perspective::set_intrinsics )
+  .def( "set_intrinsics",
```

Comment:
Is there a reason that typical PyBind11 conventions for overloading were not used? Attempting it locally, I can't seem to find an issue with using the strategy found [here](https://pybind11.readthedocs.io/en/stable/classes.html#overloaded-methods) that the rest of the bindings employ, but I may be overlooking something.

Reply:
...because that requires spelling out the class type ([twice](https://en.wikipedia.org/wiki/Don't_repeat_yourself)) and the return type?

```
        kv::overload< kv::frame_id_t, cam_sptr_t const& >(
```
versus
```
        // this line is waaaay over length with no "good" way to wrap
        static_cast< void ( kv::simple_camera_perspective::* )( kv::frame_id_t, cam_sptr_t const& ) >(
```

Is that seriously a contest?

Qt [didn't think so](https://doc.qt.io/qt-5/qtglobal.html#qOverload). (BTW, `vital::overload` is *not* copied from Qt. It's copied from a personal project that might actually be the source of `qOverload`, or at any rate IIRC predates `qOverload`.)

Reply:
Additionally, while a) it doesn't apply here, b) it is questionable practice anyway, and c) I don't believe I've ever actually (ab)used it this way, `overload` actually doesn't even require that you spell out the entire argument list; only enough to disambiguate (so e.g. if you have five arguments but the first is unique, you could write just `overload< first_t >`; other methods always require spelling out all five arguments).

Reply:
I certainly agree that the current approach is... unwieldy, and clearly this is a much better approach in terms of readability/clarity, I was just concerned I had missed something else obvious in my understanding. I apologize for not being clearer. Personally, I think it would be worth refactoring the remaining bindings to use `overload`. Thoughts?

Reply:
Nah, I think "the pybind11 way" is just the clunky way. Note too that the *code* of `overload` doesn't actually do anything (it's an identity function); it's just clever use of TAD combined with leveraging type restriction to perform guided overload resolution. (Come to think of it, I wonder if `def` can already do this, but even if so, `def<args>(name, &member)` is maybe not really an improvement over `def(name, overload<args>(&member))`.)

I am *definitely* in favor of refactoring anything that could use `overload` to do so... but that's a separate PR 🙂.

Reply:
I'll add the refactoring effort to the task list for the Python bindings, this PR looks good to go from my end!